applications/luci-app-acme/htdocs/luci-static/resources/view/acme.js

```diff
@@ -102,17 +79,345 @@ return view.extend({
 				"server must be accessible from the internet on port 80."));
 		o.optional = true;
 		o.depends("validation_method", "webroot");
+		o.default = "/var/run/acme/challenge/";
```

Comment:
With the updated acme-common, it's not just that `/var/run/acme/challenge` is the *default* for the webroot, it's that the webroot should not be user-configured at all. So we should not expose this field at all in the UI. Although we should also take care not to break existing configs (see also the comment about key size, below).

Reply:
I added the default value for a backward compatibly but also to make it clear for a user. We can leave empty. What do you think?

Reply:
Ideally, I'd like this to be a read-only field under advanced settings, with a note that it's deprecated and an option to remove it...

Reply:
I removed the default